confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java

```diff
@@ -330,12 +338,42 @@ public List<TConsensusGroupId> getRegionGroupIds(String storageGroup, TConsensus
     return result;
   }
 
-  public void serialize(ByteBuffer buffer) {
-    // TODO: Serialize ClusterSchemaInfo
+  @Override
+  public boolean takeSnapshot(File snapshotFile) throws IOException {
```

Comment:
Don't forget to add test for takeSnapshot~

Reply:
fixed.